contracts/base/GuardManager.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+import "../common/SelfAuthorized.sol";
+
+interface Guard {
+    function checkCalldata(bytes calldata data, address sender) external;
+}
+
+/// @title Fallback Manager - A contract that manages fallback calls made to this contract
+/// @author Richard Meissner - <richard@gnosis.pm>
+contract GuardManager is SelfAuthorized {
+    // keccak256("guard_manager.guard.address")
+    bytes32 internal constant GUARD_STORAGE_SLOT =
+        0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8;
+
+    function internalSetGuard(address handler) internal {
```

Comment:
Why an internal and external function?

Reply:
this was taken from the fallback handle, in case we wanted to set it with another method. But yeah not required here, therefore I will remove it before merging